CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ if(PLUGIN_HDMICEC)
     add_subdirectory(HdmiCec)
 endif()
 
```

Comment:
use PLUGIN_HDMICEC2 instead of HAS_API_HDMI_CEC_2. We have to also update bb recipe file accordingly

Reply:
Seems like both master and main branches (one of them is being used in stable2 ?) are using HAS_API_HDMI_CEC_2
	https://github.com/rdkcentral/rdkservices/blob/master/CMakeLists.txt
	https://github.com/rdkcentral/rdkservices/blob/main/CMakeLists.txt

Could you please let me know if my understanding is not correct about stable2.